README.md

```diff
@@ -358,6 +360,30 @@ resources:
 
 To disable the model validation you can use `--disableModelValidation`.
 
+## WebSocket
+`This has experimental functionality. Please report any bugs or missing features.`
+
+serverless-offline suports running a WebSocket local endpoint.
+
+Usage in order to send messages back to clients:
+
+`POST http://localhost:{port+1}/@connections/{connectionId}`
+
+Or,
+
+```
+let endpoint=event.apiGatewayUrl;
+if (!endpoint) endpoint = event.requestContext.domainName+'/'+event.requestContext.stage;
+const apiVersion='2018-11-29';
+const apiGM=new API.ApiGatewayManagementApi({ apiVersion, endpoint });
+apiGM.postToConnection({ConnectionId, Data});
+```
+Where the `event` is received in the lambda handler function.
+
+There's support for `websocketsApiRouteSelectionExpression` in it's basic form: `$request.body.x.y.z`, where the default value is `$request.body.action`.
```

Comment:
What is websocketsApiRouteSelectionExpression? (not a review change request, just a question)

Reply:
https://docs.aws.amazon.com/apigateway/latest/developerguide/apigateway-websocket-api-selection-expressions.html